src/FlatSharp.Compiler/FlatSharp.Compiler.targets

```diff
@@ -10,13 +10,17 @@
       <CompilerPath>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\..\tools\net47\FlatSharp.Compiler.exe'))</CompilerPath>
       <CompilerInvocation>$(CompilerPath)</CompilerInvocation>
     </PropertyGroup>
+    
+    <PropertyGroup>
+      <NewFlatSharpSchemaPath>"%(FlatSharpSchema.fullpath)"</NewFlatSharpSchemaPath>
```

Comment:
Can you change to QuotedSchemaPath instead of "New"? After that, I'll happily approve. Thanks for fixing this!

Reply:
OK. I'll change.